export-or-backup-using-dumpling.md

```diff
@@ -10,28 +10,38 @@ This document introduces how to use the [Dumpling](https://github.com/pingcap/du
 
 For backups of SST files (KV pairs) or backups of incremental data that are not sensitive to latency, refer to [BR](/br/backup-and-restore-tool.md). For real-time backups of incremental data, refer to [TiCDC](/ticdc/ticdc-overview.md).
 
+For detailed usage of Dumpling, use the `--help` command to view it.
```

Comment:
Why not include the English user guide (https://github.com/pingcap/dumpling/blob/master/docs/en/user-guide.md)?

Reply:
Nice catch. Updated.